notary/running_a_service.md

```diff
@@ -134,6 +134,7 @@ One way to do this would be:
 
 2. Write the following configuration file to `/tmp/server-configdir/config.json`:
 
+```json
```

Comment:
This fence is part of the numbered list, so has to be indented with 4 spaces, so that the numbered list is not interrupted (numbering otherwise start back with `1.`)

But the content inside it needs to be un-indented with 4 spaces (same indentation as the code-fence itself)

```suggestion
    ```json
```